CHANGELOG.md

```diff
@@ -5,481 +5,8 @@ BREAKING CHANGES:
-
-* A new `pg` state storage backend allows storing state in a PostgreSQL database.
-
-* The `azurerm` state storage backend supports new authentication mechanisms, custom resource manager endpoints, and HTTP proxies.
-
-* The `s3` state storage backend now supports `credential_source` in AWS configuration files, support for the new AWS regions `eu-north-1` and `ap-east-1`, and several other improvements previously made in the `aws` provider.
-
-* The `swift` state storage backend now supports locking and workspaces.
-
-### Bug Fixes
-
-Quite a few bugs were fixed indirectly as a result of improvements to the underlying language engine, so a fully-comprehensive list of fixed bugs is not possible, but some of the more commonly-encountered bugs that are fixed in this release include:
-
-* config: The conditional operator `... ? ... : ...` now works with result values of any type and only returns evaluation errors for the chosen result expression, as those familiar with this operator in other languages might expect.
-
-* config: Accept and ignore UTF-8 byte-order mark for configuration files ([#19715](https://github.com/hashicorp/terraform/issues/19715))
-
-* config: When using a splat expression like `aws_instance.foo.*.id`, the addition of a new instance to the set (whose `id` is therefore not known until after apply) will no longer cause all of the other ids in the resulting list to appear unknown.
-
-* config: The `jsonencode` function now preserves the types of values passed to it, even inside nested structures, whereas before it had a tendency to convert primitive-typed values to string representations.
-
-* config: The `format` and `formatlist` functions now attempt automatic type conversions when the given values do not match the "verbs" in the format string, rather than producing a result with error placeholders in it.
-
-* config: Assigning a list containing one or more unknown values to an argument expecting a list no longer produces the incorrect error message "should be a list", because Terraform is now able to track the individual elements as being unknown rather than the list as a whole, and to track the type of each unknown value. (This also avoids any need to place seemingly-redundant list brackets around values that are already lists, which would now be interpreted as a list of lists.)
-
-* cli: When `create_before_destroy` is enabled for a resource, replacement actions are reflected correctly in rendered plans as `+/-` rather than `-/+`, and described as such in the UI messages.
-
-* core: Various root causes of the "diffs didn't match during apply" class of error are now checked at their source, allowing Terraform to either avoid the problem occurring altogether (ideally) or to provide a more actionable error message to help with reporting, finding, and fixing the bug.
-
----
+For information on v0.12, please see [the v0.12 branch changelog](https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md).
```

Comment:
I think I would've just conflated this and the following paragraph into a single paragraph just linking to the v0.12 changelog, making a "linked-list" sort of structure, but I don't feel super-strongly about it... maybe if we're going to preserve links to all historical versions over time it would be better to pivot to a list format now, though:

> For information on prior major releases, see their changelogs:
> * [v0.12](https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md)
> * [v0.11 and earlier](https://github.com/hashicorp/terraform/blob/v0.11/CHANGELOG.md)

...then it's clearer what we need to do when `master` becomes v0.14 later.

Reply:
That's great, thanks! Updated :D